docs/how-to-guides/create-backup.md

```diff
@@ -3,149 +3,137 @@ id: create-backup
--rw-r----- 1 memgraph memgraph 17064381 Mar 25 13:53 20220325125308366007_timestamp_3380
-```
+  If you've just created a snapshot file there is no need to backup WAL files. 
 
-**4.** Copy a file from the `snapshots` directory to the backup folder:
+  To help copying the files from the Docker container, check out the [Working with
+  docker
+  guide](/how-to-guides/work-with-docker.md#how-to-copy-files-from-and-to-a-docker-container).
 
-```bash
-cp /var/lib/memgraph/snapshots/20220325125308366007_timestamp_3380 ~/backup/
-```
+4. **Unlock the data directory**
 
-</details>
+  Run the following query in `mgconsole` or Memgraph Lab to unlock the
+  directory:
 
-If you need help copying the files from the Docker container, check out the
-[Working with docker
-guide](/how-to-guides/work-with-docker.md#how-to-copy-files-from-and-to-a-docker-container).
+  ```cypher
+  UNLOCK DATA DIRECTORY;
+  ```
 
-Then, run the following query in `mgconsole` or Memgraph Lab to unlock the
-directory:
-
-```cypher
-UNLOCK DATA DIRECTORY;
-```
-
-Memgraph will delete the files which should have been deleted before locking and
-allow any future deletion of the durability files.
+  Memgraph will delete the files which should have been deleted before locking and
+  allow any future deletion of the durability files.
 
 ## Restore data
 
-To restore data from a backup: 
-
-### 1. Lock the data directory
-
-To disable deletions of snapshot or WAL files in the data directory, run the following query in
-`mgconsole` or Memgraph Lab:
-
-```cypher
-LOCK DATA DIRECTORY;
-```
-
-### 2. Copy the snapshot into the data directory
-
-Copy the data directory or, empty the `snapshots` and `wal` directories, then
-copy a single WAL or snapshot file into the respective folders.
-
-If you need help copying the files from the Docker container, check out the
-[Working with docker
-guide](/how-to-guides/work-with-docker.md#how-to-copy-files-from-and-to-a-docker-container).
-
-### 3. Restart the instance
-
-By restarting the instance, Memgraph should restore the data from the files in
-the data directory. 
-
-Be sure to restart the instance before Memgraph automatically creates a new
-periodic snapshot because upon restart it might use that newer snapshot which is
-not the data you want to load. 
\ No newline at end of file
+To restore data from a backup
+
+<Tabs
+  groupId="backup"
+  defaultValue="docker"
+  values={[
+    {label: 'Docker 🐳', value: 'docker'},
+    {label: 'Linux', value: 'linux'}
+  ]}>
+<TabItem value='docker'>
+
+
+1. Empty the `wal` directory
+
+  If you want to restore data only from the snapshot file, ensure that the
+  `wal` directory is empty:
+
+     - Find the container ID using a `docker ps` command, then enter the container using:
+
+    ```
+    docker exec -it CONTAINER_ID bash
+    ```
+     - Position yourself in the `/var/lib/memgraph/wal` directory and `rm *`
+
+1. Stop the instance using `docker stop CONTAINER_ID`
```

Comment:
Not sure why it is relevant when you are c/p the whole directory with the one you have on disk... (this is a how-to, not a reference guide so only info relevant to this process are written)
So whatever i have in the container will be replaced with my back-up dir